GCC versions before 4.5 don't accept -std=c90, which is just a synonym
for -std=c89.
--- zlibWrapper/Makefile.orig	2019-11-05 04:54:32.000000000 +1100
+++ zlibWrapper/Makefile	2019-11-07 21:25:49.000000000 +1100
@@ -20,7 +20,7 @@
 
 CPPFLAGS += -DXXH_NAMESPACE=ZSTD_ -I$(ZLIB_PATH) -I$(PROGRAMS_PATH)       \
             -I$(ZSTDLIBDIR) -I$(ZSTDLIBDIR)/common -I$(ZLIBWRAPPER_PATH)
-STDFLAGS  = -std=c90 -pedantic -Wno-long-long -Wno-variadic-macros -Wc++-compat \
+STDFLAGS  = -std=c89 -pedantic -Wno-long-long -Wno-variadic-macros -Wc++-compat \
             -DNO_snprintf -DNO_vsnprintf  # strict ISO C90 is missing these prototypes
 DEBUGFLAGS= -Wall -Wextra -Wcast-qual -Wcast-align -Wshadow -Wswitch-enum \
             -Wdeclaration-after-statement -Wstrict-prototypes -Wundef     \
